fred2/orienteditor.cpp

```diff
@@ -182,9 +182,27 @@ BOOL orient_editor::OnInitDialog()
 	return TRUE;
 }
 
-bool orient_editor::close(float val, const CString &str)
+bool orient_editor::close(float val, const CString &input_str)
 {
-	float diff = val - convert(str);
+	CString str;
+	str.Format("%.01f", val);
```

Comment:
A comment explaining why we're doing this would be nice IMO, maybe even citing the related GH Issue.

Reply:
Also, since we use this call below, perhaps a function like
```
CString normalize_float(float value) const;
```
(or equivalent if MFC doesn't like handling `CString` objects that way) but maybe that's overkill.

Reply:
Ok.